modules/gpio.sh

```diff
@@ -0,0 +1,51 @@
+function gpio {
+  pinout | sed -n -e '/,-/,/|V|/ p'
+  echo
+  pinout | grep -E "3V3|GPIO"
+}
+
+function gpio_help {
+  echo
+  echo "Usage: $BASENAME gpio"
+  echo
+  echo "Displays picture model of raspberry pi and corresponding GPIO ports"
+  echo
+  echo "Example:"
+  echo "  $BASENAME gpio"
+  echo "      on"
```

Comment:
Is this `on` necessary?

Reply:
nope, stayed in accidentally from a copy/paste